ci/asan_tests/ray-project/cluster.yaml

```diff
@@ -0,0 +1,49 @@
+# This file is generated by `ray project create`.
+
+# A unique identifier for the head node and workers of this cluster.
+cluster_name: asan_python_tests
+
+# The maximum number of workers nodes to launch in addition to the head
+# node. This takes precedence over min_workers. min_workers defaults to 0.
+max_workers: 1
```

Comment:
Why not having deterministic number of workers? Is it intended?

Reply:
nope. this is just the default generated by `ray project init`